junit/junit_test.go

```diff
@@ -72,6 +77,12 @@ func TestCreateFromReport(t *testing.T) {
 						Time:      "0.000",
 						SystemOut: &Output{Data: "ok"},
 					},
+					{
+						Name:      "TestEscapeOutput",
+						Classname: "package/name",
+						Time:      "0.000",
+						SystemOut: &Output{Data: "�\v\f \t\\"},
```

Comment:
`\v` and `\f` are also considered illegal characters, they should also map to `\uFFFD`.